src/render/attrs/index.js

```diff
@@ -10,6 +10,42 @@ const attrsNotToRender = {
   dangerouslySetInnerHTML: true
 };
 
+const attrsWithExplicitBoolValue = {
+  "aria-expanded": true,
+  "aria-haspopup": true
+};
```

Comment:
Here I am, I took some time to make a working POC: 

https://gist.github.com/mijamo/9ece5c764071f343885401074e078182

The `style`property still needs to be handled separately, as seen in Fiber for instance (https://github.com/facebook/react/blob/b1768b5a48d1f82e4ef4150e0036c5f846d3758a/src/renderers/dom/fiber/ReactDOMFiberComponent.js#L335 ) but that simple POC should be able to handle all the other combinations of attr/values supported by React, as it corresponds to the internals of React as of today :)

Do not hesitate to ask if you need help integrating that in rapscallion, I can take a few hours from work to do that, we are very interested in rapscallion!